cmd/ols-load-generator/types.go

```diff
@@ -1,14 +1,19 @@
 package main
 
+import (
+	"time"
+)
+
 // Type to store the test config.
 type TestConfig struct {
-	Rps        int      `json:"rps"`
-	Host       string   `json:"host"`
-	HitSize    int      `json:"hitsize"`
-	AuthToken  string   `json:"-"`
-	Uuid       string   `json:"uuid"`
-	ESHost     string   `json:"eshost"`
-	ESIndex    string   `json:"esindex"`
-	MetricStep int      `json:"metricstep"`
-	Profiles   []string `json:"profiles"`
+	Rps        int           `json:"rps"`
```

Comment:
Here we're keeping the `Prs` field while it's gone from the `Document`.

Reply:
Updated.